src/main/kotlin/bd/clinic/modules/order/OrderService.kt

```diff
@@ -30,11 +41,39 @@ class OrderService(
     fun addPatientOrder(order: OrderDTO): OrderDTO {
         checkIfOrderNumberIsUnique(order.orderNumber)
         val patient = patientService.findPatientOrThrow(order.patientId)
+        sendPostRequestToAllLaboratories(order)
+
         return OrderDTO(orderRepository.save(order.toOrderEntity(patient)))
     }
 
+    private fun sendPostRequestToAllLaboratories(orderDTO: OrderDTO) {
```

Comment:
nazwalbym to jakos zeby bylo wiadomo co sie dzieje w praktyce, typu zapisz badania w laboratoriach czy cos. tak samo to nizej

Reply:
Ale tu wysyłasz request, laby sobie same zapisują, więc nie wiem, czy nazwa "zapisz badania" jakoś nie zaciemnia tego

Reply:
A dobra, bo to jest orderSevice, to fakt